### PR TITLE
Document Organizations

### PR DESCRIPTION
**Pull request by GuGuss**

### CODE REVIEW

**Comment by chadwcarlson** [comment]
Leaving a reminder here to remove/update Vouchers wrt Orgs.  @GuGuss @MariadeAnton

**Comment by GuGuss** [comment]
> Leaving a reminder here to remove/update Vouchers wrt Orgs. @GuGuss @MariadeAnton

Done ✅

File: docs/src/administration/organizations.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+For example, the following command would invite `alice@example.com` with the **Billing**, **Plans** and **Projects create** permissions to the `my-organization-name` organization.
+
+```bash
+platform organization:user:add alice@example.com --org=my-organization-name --permission=billing,plans,projects:create
+```
+
+After inviting `alice@example.com`, Alice will receive an invitation email asking to confirm her details and optionally, register for a Platform.sh account.
+
+To update Alice permissions in your organization, simply run:
```
Comment (pjcdawkins) [inline docs/src/administration/organizations.md:63]:
```suggestion
To update Alice's permissions in your organization, run:
```

File: docs/src/administration/organizations.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+title: "Organizations"
+weight: -1
+sidebarTitle: "Organizations"
+description: |
+  Organizations allow to manage Platform.sh projects, users and billing.
```
Comment (chadwcarlson) [inline docs/src/administration/organizations.md:6]:
```suggestion
  Organizations allow you to manage Platform.sh projects, users and billing.
```

File: docs/src/administration/organizations.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+---
+title: "Organizations"
+weight: -1
+sidebarTitle: "Organizations"
+description: |
+  Organizations allow to manage your Platform.sh projects, users and billing.
```
Comment (pjcdawkins) [inline docs/src/administration/organizations.md:6]:
```suggestion
  Organizations allow you to manage your Platform.sh projects, users and billing.
```
Chad's suggestion, updated

File: docs/src/administration/organizations.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+You can use the Platform.sh command line interface to manage your organizations.
+
+Available commands (get the full list with 'platform list organization')
+
+```
+Available commands for the "organization" namespace:
+  organization:billing:address                                  View or change an organization's billing address
+  organization:billing:profile                                  View or change an organization's billing profile
+  organization:create                                           Create a new organization
```
Comment (pjcdawkins) [inline docs/src/administration/organizations.md:42]:
```suggestion
```

**Pull request merged**